### PR TITLE
Create "handshake" topic

### PR DESCRIPTION
**Pull request by angrymouse**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

>I've added handshake topic
---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by angrymouse** [comment]
2000 closed PRs 🎉 
image

File: topics/handshake/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+display_name: Handshake
+github_url: https://github.com/handshake-org
+logo: handshake.png
+released: 2018
+short_description: Handshake is a decentralized naming and certificate authority 
+topic: haskell
```
[the remark this answers is not in the document]
Comment (angrymouse) [reply topics/handshake/index.md:7]: Thanks for correction

File: topics/handshake/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+url: https://www.handshake.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Alternative_DNS_root
```
[the remark this answers is not in the document]
Comment (angrymouse) [reply topics/handshake/index.md:9]: Thank you

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

File: topics/handshake/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+display_name: Handshake
+github_url: https://github.com/handshake-org
+logo: handshake.png
+released: 2018
+short_description: Handshake is a decentralized naming and certificate authority
```
Comment (kenyonj) [inline topics/handshake/index.md:6]:
```suggestion
short_description: Handshake is a decentralized naming and certificate authority.
```

**Pull request merged**